cypress/integration/1-getting-started/Local tests/topbar/admin/1_Helper_dropdown_button_functionalities.spec.js

```diff
@@ -0,0 +1,38 @@
+/// <reference types="cypress" />
+const baseURL = Cypress.env("baseURL");
+// Before running this test first run the Server test 2 in order to create a user so you can log in.
+describe("Test helper dropdown functionalities", () => {
+  context("This is a function for visiting the log in page", () => {
+    it("should be able to visit the login page", () => {
```

Comment:
why not merge this test and the next into one? Here you're only visiting the login page.